public/js/reader.js

```diff
@@ -438,21 +424,24 @@ Reader.goToPage = function (page) {
         if (img1.naturalWidth > img1.naturalHeight || img2.naturalWidth > img2.naturalHeight) {
             // Depending on whether we were going forward or backward, display img1 or img2
             $("#img").attr("src", Reader.previousPage > Reader.currentPage ? img2.src : img1.src);
-            $("#img_pre").attr("src", "");
+            $("#img_doublepage").attr("src", "");
+            $("a#display").removeClass("double-mode");
```

Comment:
It'd be better to move this block:  
```
$("#img_doublepage").attr("src", "");
$("#display").removeClass("double-mode");
``` 
before the `if` statement on line 420, so that we only have to write it once 👍 (and not twice in both branches)